_maps/configs/chapel.json

```
@@ -0,0 +1,13 @@
+{
+	"map_name": "Shepherd-Class Space Monastery",
+	"map_short_name": "Space Church",
```

please make the short name the map's ship class (in this case `Shepherd-class`) thanks :)

---

please make single-line edit requests a code suggestion
```suggestion
	"map_short_name": "Shepherd-class",
```
thanks :)